webapp/src/webusb.tsx

```diff
@@ -470,18 +470,67 @@ function theme() {
 export function renderUnpairDialog() {
     const boardName = getBoardName();
     const header = lf("How to unpair your {0}", boardName);
-    const jsx = <div><p>
-        {lf("You can unpair your {0} if the WebUSB download is malfunctioning. Click on the lock icon and uncheck your device.", boardName)}
-    </p>
-        <img className="ui image centered medium" src={"./static/webusb/unpair.gif"} alt={lf("A gif showing how to unpair the {0}", boardName)} />
+    const unpairImg = theme().browserUnpairImage;
+    const jsx = <div>
+        <p>
+            {lf("You can unpair your {0} if the WebUSB download is malfunctioning.", boardName)}
+        </p>
+        <p>
+            {lf("Click on the icon on the left side of the address bar and uncheck your device.")}
```

Comment:
Should it be "unmount your device" or will it actually be a check box?

Reply:
It's a bit different depending on both browser and version, e.g. an relatively recent version of chrome I have looks like this: 
![image](https://github.com/microsoft/pxt/assets/5615930/6619175e-cee5-4585-8f5d-4629268cfa67)
with chrome canary looking similar but with a little more emphasis on reset
![image](https://github.com/microsoft/pxt/assets/5615930/2444137e-ff00-40a8-a873-389e513c8118)

I mostly left the wording how it originally was there since this would typically only be shown to people on older chrome installs anyways, and I'm assuming 'check' was closer to what the dialog looked like when it was originally written (chrome 101 was apr 2022)